apps/frontend/src/pages/[type]/[id]/gallery.vue

```diff
@@ -716,6 +716,9 @@ export default defineNuxtComponent({
 
       p {
         margin: 0 0 0.5rem 0;
+        max-width: 250px;
+        text-overflow: ellipsis;
+        overflow: hidden;
```

Comment:
```suggestion
        overflow-wrap: break-word;
```

We don't want to hide the info, does this work better?

Reply:
(and get rid of the `text-overflow: ellipsis;`

Reply:
hell yeah, completely forgot about `overflow-wrap` existing lol